plugins/modules/zos_mvs_raw.py

```diff
@@ -3090,6 +3102,42 @@ def get_content(formatted_name, binary=False, from_encoding=None, to_encoding=No
         return stdout
 
 
+def modify_contents(contents):
+    """Return the content of dd_input to a valid form for a JCL program.
+
+    Args:
+        contents (str or list): The string or list with the program.
+
+    Returns:
+        contents: The content in a proper multi line str.
+    """
+    if not isinstance(contents, list):
+        contents = list(contents.split("\n"))
+    contents = prepend_spaces(contents)
+    contents = "\n".join(contents)
+    return contents
+
+
+def prepend_spaces(lines):
+    """Return the array with two spaces at the beggining.
+
+    Args:
+        lines (list): The list with a line of a program.
+
+    Returns:
+        new_lines: The list in a proper two spaces and the code.
+    """
+    module = AnsibleModuleHelper(argument_spec={})
+    for index, line in enumerate(lines):
+        if len(line) > 0:
```

Comment:
I think we have a few gaps; the logic here checks if the line is greater than 0 which is good, next it checks if the line is greater than 78 which is going to lead to problems if a user gives us content that is 80 chars long and includes the 2 blanks, eg 2 blank chars + 78 non-blanks = 80. 

In the case above, I believe what we should be doing is checking the line is >0, then if the line is <=80 continue, if so, check that positions 0 and 1 are blank , this allows a user to provide us lines that are 80 in length that includes 2 blanks. 

I believe this logic I am proposing will also work for a YAML block indicator e.g. `content: | 4` as this will produce a line with 4 blanks + some non-blank chars up to 80. We would only want to check that the line length is <= 80 and postions 0 and 1 are blank.

If we find a line that is <=80 and has no blanks, we can prepend the blanks in columns 1 and 2 but we also need to check that we don't create a line linger than 80 chars after we add the spaces, if we do we need an appropriate message to the user.